### DISCUSSION TITLE
TrustWallet on Android issue

### DISCUSSION THREAD

**Discussion by pmo-van-018** [community/community#72954] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

TrustWallet on Android always add the `blob:` prefix into my script src, so It make the url become invalid. E.g: my url was https://challenge.cloudflare.com => blob:https://challenge....
IOS was ok but Android really bad.

**Comment by mhaemnn** [comment]
Check your HTML or JavaScript source code to ensure that on the Android platform, there is no addition of "blob:" to the src attribute of the script. Check if there are any differences in implementation between Android and iOS platforms.

**Reply by pmo-van-018** [reply]
There is no addition of "blob:" to the src attribute of the script. I Have double checked

**Comment by mhaemnn** [comment]
Check TrustWallet Version:

Make sure that you are using the latest version of TrustWallet. There may be related issues that have been fixed in the latest version.

**Reply by pmo-van-018** [reply]
I have updated lastest version (8.1.1)

**Comment by pmo-van-018** [comment]
Anyone have met this issue?
